src/solc_0.7/common/Game/GameToken.sol

```diff
@@ -0,0 +1,381 @@
+//SPDX-License-Identifier: MIT
```

Comment:
the common folder is for things implementation need
I would put the GameTOken in src/solc_0.7/GameToken folder

Reply:
fixed.